projects/vdk-plugins/vdk-impala/.plugin-ci.yml

```diff
@@ -1,7 +1,6 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-image: "python:3.7"
```

Comment:
nit: those are redundant, since overwritten? what's the effective python version definition?